client/css/style.css

```diff
@@ -855,13 +854,13 @@ kbd {
 	left: 0;
 	right: 0;
 	top: 50%;
-	border-top: 1px solid #e74c3c;
+	border-top: 1px solid rgba(231, 76, 60, .5);
```

Comment:
What @MaxLeiter said. If I don't do this *on the date marker*, the tooltip is half-transparent.
The fact that I changed the unread marker as well was 1) to be consistent, 2) because there were some shared CSS so I had to either do that, or do a mixed ugly version of it, not worth it. Better to fix completely (it's also nicer for the themes).